### PR TITLE
Failure to upgrade

### PR DESCRIPTION
**Pull request by coreydaley**
https://bugzilla.redhat.com/show_bug.cgi

### CODE REVIEW

**Comment by adambkaplan** [comment]
@coreydaley I tend to prefer that all vendor code updates should happen in the bump(*) commit, and any change thereafter has its own commit(s) - even if that means the bump results in broken/non-compiling code. It helps keep the history clean.

**Comment by coreydaley** [comment]
@adambkaplan fixed, must have over-squashed before my last push.

**Comment by adambkaplan** [comment]
/hold

per @mfojtik

File: glide.lock [hunk 1]
```diff
@@ -474,7 +474,7 @@ imports:
   - storage/v1alpha1
   - storage/v1beta1
 - name: k8s.io/apiextensions-apiserver
-  version: 63bd5e1c39483f0ef7fd275794d19a7cba19b764
+  version: 36be1826c980fa7ca82d91645e06e3157f159322
```
Comment (adambkaplan) [inline glide.lock]: @coreydaley We should pin this (and any other kube deps) to a 1.12 branch. 

Let's split this out into a separate PR and update this PR to only contain the changes around progress reporting in the sync controller.

Comment (coreydaley) [reply glide.lock]: @adambkaplan Aren't they using 1.12 in the glide.yaml file? https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/glide.yaml ?

**Comment by coreydaley** [comment]
/retest

level=error msg="\t* aws_vpc.new_vpc: Error creating VPC: VpcLimitExceeded: The maximum number of VPCs has been reached."

**Comment by adambkaplan** [comment]
@coreydaley the openshift-apiserver-operator has unit test patterns we can take advantage of for this change: https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00_test.go#L33-L173

**Comment by adambkaplan** [comment]
/hold cancel

We aren't bumping deps in this PR any more
